Dockerfile.d/etc_containerd_config.toml

```diff
@@ -5,3 +5,9 @@ version = 2
   [proxy_plugins.stargz]
     type = "snapshot"
     address = "/run/containerd-stargz-grpc/containerd-stargz-grpc.sock"
+
+# Enable soci snapshotter
+[proxy_plugins]
```

Comment:
To add soci snapshotter, you should just add
```
  [proxy_plugins.soci]
    type = "snapshot"
    address = "/run/soci-snapshotter-grpc/soci-snapshotter-grpc.sock"
```
under same `[proxy_plugins]`

You can't have two `proxy_plugins` in the containerd config file.